### PR TITLE
fix(@ngtools/webpack): Lazy load from node_modules

### PR DESCRIPTION
**Pull request by pedroapy**
Based on code of PR #6609. Here's a possible solution for issue #6373

### CODE REVIEW

File: packages/@ngtools/webpack/src/refactor.ts [hunk 1]
```diff
@@ -205,6 +206,9 @@ export class TypeScriptFileRefactor {
   }
 
   transpile(compilerOptions: ts.CompilerOptions): TranspileOutput {
+    if (this._fileName.endsWith('.d.ts') && this._fileName.indexOf('node_modules') > -1) {
```
Comment (hansl) [inline packages/@ngtools/webpack/src/refactor.ts]: You're not verifying that a path component is `node_modules`, but just that `node_modules` is a substring. Could you use something like `this._fileName.split(path.sep).indexOf('node_modules') >= 0`?

Comment (pedroapy) [reply packages/@ngtools/webpack/src/refactor.ts]: Thank you, you're right, I just fixed it.

**Comment by rhinoandre** [comment]
So, what is missing to this PR be approved?

**Comment by choeller** [comment]
We are also very interested in this PR. @hansl As your comment was take care of, Could this be merged?

**Comment by mmayors** [comment]
Glad to see this PR has been opened, it would be really nice to support this out of the box without additional Webpack config / loaders.

**Comment by yanqui92** [comment]
Hi, I'm working with multiple custom libraries and this PR would be very useful for me and many others.

**Comment by Mani2693** [comment]
This PR will be so helpful if it fix the issue, Can we have it with next build 

Thanks for the help

**Comment by dotansimha** [comment]
@pedroapy @hansl any chance we can get it merged? :(

**Comment by yanqui92** [comment]
@filipesilva any chance we can get it merged?

**Comment by clydin** [comment]
Thank you for the contribution.  However, this is being closed as it is outdated and no longer applies to the current code base.

**Comment by mmayors** [comment]
@clydin may I ask why it's no longer applicable?  Are there now other ways to lazy load NgModules that live outside the app src?  (i.e. inside an NPM package / node_modules)

**Comment by clydin** [comment]
The changes in question were made to a file that no longer exists within the code base as it was used only for Angular version less then 5.  In addition the package in question has been moved to this repository: https://github.com/angular/devkit

**Pull request closed without merging**